page-index/page-index.js

```diff
@@ -133,10 +133,15 @@ const pageIndex = [
                 href: "setup-multi-node-basic",
                 children: [
                     {
-                        Title: "Configuring authentication",
+                        Title: "Self-managed Multi-Node Setup",
```

Comment:
This is a mouthful.  Maybe just "Securing Access" or something? Or just keep as "Configuring access" or "Configuring authentication"?  Or "Advanced setup" ? Reader knows it's about multi-node already given hierarchy"

Reply:
None of these really seem to quite capture the content, but "Advanced Setup" does seem like an improvement over the current title.  I'll just go with that.